packages/rolldown/src/utils/compose-js-plugins.ts

```diff
@@ -59,14 +54,6 @@ function createComposedPlugin(plugins: Plugin[]): Plugin {
           }
           break
         }
-        case 'resolveId': {
```

Comment:
We could keep this as long as we will skip plugin that has resolveId hook previously.

Reply:
Looks like a bug, can I comment out?

Reply:
Ok, looks like I can't due to exaustive check